Libraries/Components/View/View.js

```diff
@@ -258,6 +258,35 @@ const View = React.createClass({
      * Used to locate this view in end-to-end tests.
      *
      * > This disables the 'layout-only view removal' optimization for this view!
+     *
+     * ### Android Specifics
+     *
+     * By default this value is passed to the underlying View's
+     * [setTag](https://developer.android.com/reference/android/view/View.html#setTag(java.lang.Object))
+     * method.  Very few testing frameworks support looking up
+     * a View by tag, let alone `uiautomatorviewer` which only supports `content-description`, `resource-id`, `text`, and `XPath`.
+     *
+     * While `react-native` does *not* utilize XML based layouts for android Views it
+     * is still possible to support [android:id](https://developer.android.com/reference/android/view/View.html#attr_android:id)
+     * to the underlying View in order to support
+     * [findViewById](https://developer.android.com/reference/android/app/Activity.html#findViewById(int)).
+     *
+     * This is achieved by:
+     *
+     * 1. Defining a resource id in your android project's `res` folder (typically at
+     *    `./android/app/src/main/res/values/ids.xml`).
+     *
+     * 1. Adding your resource ids to `ids.xml` E.G.
```

Comment:
Markdown allows you to use any number here for ordered lists and it handles it.  I find it easier to maintain, but will change it if you'd like me to.

Here's what it looks like:
![docs](https://cloud.githubusercontent.com/assets/2605980/18795590/203f308c-817a-11e6-8b2b-a4f1c6b5535b.png)

Thanks for sharing how to run the docs!